aiohttp/payload.py

```diff
@@ -87,12 +101,23 @@ def register(self, factory, type, *, order=Order.normal):
 
 class Payload(ABC):
 
-    _size = None
-    _headers = None
-    _content_type = 'application/octet-stream'
-
-    def __init__(self, value, *, headers=None, content_type=sentinel,
-                 filename=None, encoding=None, **kwargs):
+    _size = None  # type: Optional[float]
```

Comment:
A bit late for the party, but why size is typed as float? This value is used for Content-Length header which defined number of bytes of payload size - it's hard to pass over HTTP some fraction of a byte. What was the case for float type?

Reply:
https://github.com/aio-libs/aiohttp/pull/3484